src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray_1.Builder.cs

```diff
@@ -613,21 +613,7 @@ public int LastIndexOf(T item, int startIndex, int count, IEqualityComparer<T> e
             /// </summary>
             public void Reverse()
             {
-                // The non-generic Array.Reverse is not used because it does not perform
-                // well for non-primitive value types.
-                // If/when a generic Array.Reverse<T> becomes available, the below code
-                // can be deleted and replaced with a call to Array.Reverse<T>.
```

Comment:
If this is going to be under a netcoreapp ifdef then I'll revert this because the point of the change was to remove, not add code.